src/test/java/de/retest/web/selenium/css/HasTest.java

```diff
@@ -74,8 +74,15 @@ void should_match_attribute_with_value_containing_substring( final String select
 		);
 	}
 
-	@Test
-	void should_match_attribute_with_value_containing_word() throws Exception {
+	@MethodSource( "attributeMatchersContainingWord" )
+	@ParameterizedTest( name = "[{index}] using selector ''{0}'', to match ''{1}'', expecting {2}" )
+	void should_match_attribute_with_value_containing_word( final String selector, final String matchingValue,

```

Comment:
`matchingValue` no longer describes only the matching value. I would recommend to rename it.

Reply:
I changes it to simply say `value`.